public/locales/en/translation.json

```
@@ -0,0 +1,47 @@
+{
+  "Common": {
+    "Showing": "Mostrando",
+    "to": "to",
+    "of": "of",
+    "Active filters": "Active filters",
+    "Reset filters": "Reset filters",
+    "Filters": "Filters",
+    "Close menu": "Close menu",
+    "Next page":"Next page",
+    "Previous page":"Previous page"
+  },
+  "Login_page": {
```

```suggestion
  "Login": {
```
I would drop the "_page" suffix. Same goes for the other pages and Italian file